docs/user_api/index.rst

```diff
@@ -175,6 +176,10 @@ Attributes
    Grid.Mesh2_edge_nodes
    Grid.Mesh2_face_edges
    Grid.antimeridian_face_indices
+   Grid.corner_node_balltree
+   Grid.center_node_balltree
+   Grid.corner_node_kdtree
+   Grid.center_node_kdtree
```

Comment:
@aaronzedwick 

I think these are causing your issue, we don't actually have these attributes.